server/modules/taxonomy/api/taxonomyApi.js

```diff
@@ -147,13 +151,18 @@ export const init = app => {
       try {
         const { surveyId, taxonomyUuid, draft } = Request.getParams(req)
 
-        Response.setContentTypeFile(res, `taxonomy_${taxonomyUuid}.csv`, null, Response.contentTypes.csv)
+        const survey = await SurveyService.fetchSurveyById(surveyId, draft)
+        const surveyName = Survey.getName(Survey.getSurveyInfo(survey))
+        const taxonomy = await TaxonomyService.fetchTaxonomyByUuid(surveyId, taxonomyUuid, draft)
+        const timestamp = DateUtils.format(Date.now(), 'yyyy-MM-dd_HH-mm-ss')
```

Comment:
Should there be a function in DateUtils to format timestamps in that way so we can use it elsewhere?!

Reply:
Sure, it can be called ```DateUtils.nowFormatDefault()``` ?

Reply:
ok